src/lib/plugins/live-cell/live-cell.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Input, OnInit, SimpleChange, ChangeDetectionStrategy } from '@angular/core';
+import { TemplateHostService } from '../../template/template-host.service';
+import { TdCoreDirective } from 'lib/main/core/body/td-core.directive';
+
+@Component({
+	selector: 'q-grid-live-cell',
+	templateUrl: './live-cell.tpl.html',
+	providers: [TemplateHostService],
+	changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class LiveCellComponent implements OnInit {
+
+	@Input() cell: TdCoreDirective;
+	@Input() duration = 500;
+
+	difference = 0;
+
+	constructor() {
+	}
+
+	ngOnInit() {
+		if (!this.cell.changes) {
```

Comment:
I believe we can not enter here if there is no changes, get rid of this check please.
Other way is to throw exception.